config/routes.rb

```diff
@@ -52,6 +52,7 @@
       end
     end
     # (Albert Wang): Need to factor the above promo_registrations, as they should be in Publishers::PromoRegistrationsController rather than in the PromoRegistrationsController
+    resources :policy_agreements, controller: 'publishers/policy_agreements', only: [:new, :create]
```

Comment:
Consider moving this up one line so that the comment on the line above it makes sense